docker/docker-compose-release.yml

```diff
@@ -7,11 +7,10 @@ services:
       dockerfile: docker/Dockerfile.release
     command: /app/coordinator -c /app/config.toml
     volumes:
-      - ${PWD}/../configs/docker-release.toml:/app/config.toml
```

Comment:
I'm happy to revert that change. We just have to make sure that we document in which folder the user must be in order to start docker-compose. Normally I start docker-compose in the root of the repo via the command: `docker-compose -f docker/docker-compose.yml up`